pynamodb/expressions/condition.py

```diff
@@ -0,0 +1,85 @@
+from pynamodb.expressions.util import get_value_placeholder, substitute_names
+
+
+class Path(object):
```

Comment:
Next change set is going to attempt to address support for 'dotted' attributes and better support for document paths. Will include in that PR since I'm not sure yet how this class with change.